docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`1015` Rotki now lets the user manually refresh and save their balances, even if the balance save frequency has not lapsed. This functionality is accessible through the Save Indicator (floppy disk icon on the app bar).
```

Comment:
Can you use the terminology: "Take a snapshot of their balances" somewhere in here?

Reply:
Done.